### DISCUSSION TITLE
My Revise.jl not working with Pluto.jl anymore

### DISCUSSION THREAD

**Discussion by zhangcx93** [fonsp/Pluto.jl#1223] [Questions about using Pluto] [answered]
I've been prototyping with revise and pluto all the time, but since maybe a recent update, any Revise.jl doesn't seem to work anymore, it just has no effect, no error message either.
I tried doing it in a empty project, it still doesn't work. But in REPL, revise is still working.

Is there any new way of using Revise.jl with Pluto.jl is added? Or it's a bug?
It was working perfectly, I just add using Revise in startup.jl, and start Pluto.

**Comment by zhangcx93** [comment]
MME:
```
] generate TestPluto
# go to the dir
] activate .
] add Pluto
using Pluto
Pluto.run()
```
You run this in REPL or Pluto:
```julia
using Revise
using TestPluto

TestPluto.greet()
```

and edit the greet() in src/TestPluto.jl

only the one in the REPL has response to the change, but the Pluto didn't apply the change.
It was working before since I've been rely on the reactive edit of my module dev.

**Reply by lungben** [reply]
You have to put `using Revise` inside the Pluto notebook so that it is executed before using the package to be tracked.
```
begin
using Revise
using TestPluto
end
```

**Reply by zhangcx93** [reply]
I tried this but it's not working in current version.
Before this I can put it in startup.jl, or in a seperate cell or in the same cell, but in current version none of these works.

**Comment by fonsp** [comment, marked as answer]
It's already fixed, but not yet released: https://github.com/fonsp/Pluto.jl/issues/1212

**Reply by fonsp** [reply]
Has been released as 0.14.8 now
